scripts/common.sh

```diff
@@ -312,10 +398,246 @@ function CONFIRM() {
 }
 
 #-------------------------------------------------------------------------------
-# Dependency checks
+function merge_homebrew() {
+  CONFIRM "The deploy robot should have sent a pull request to https://github.com/kong/homebrew-kong/pulls . " \
+          "Make sure it gets approved and merged. Press Enter when done"
+  SUCCESS "Homebrew PR merged. Proceeding!"
+}
+
 #-------------------------------------------------------------------------------
+function merge_pongo() {
+  CONFIRM "The deploy robot should have sent a pull request to https://github.com/kong/kong-pongo/pulls . " \
+          "Make sure it gets approved and merged."
+  SUCCESS "Pongo PR merged. Proceeding!"
+}
+
+#-------------------------------------------------------------------------------
+function merge_vagrant() {
+  CONFIRM "The release robot should have sent a PR to the kong-vagrant repo: https://github.com/Kong/kong-vagrant . " \
+          "Make sure it gets approved and merged. Press Enter when done"
+  SUCCESS "Vagrant PR merged. Proceeding!"
+}
+
+#-------------------------------------------------------------------------------
+function docs_pr() {
+  branch=$1
+
+  if [ -d ../docs.konghq.com ]
+  then
+     cd ../docs.konghq.com
+  else
+     cd ..
+     git clone git@github.com:Kong/docs.konghq.com.git
+     cd docs.konghq.com
+  fi
+  git checkout main
+  git pull
+  git checkout -B "$branch"
+  bump_docs_kong_versions
+
+  git diff
+
+  CONFIRM "If everything looks all right, press Enter to commit and send a PR to git@github.com:Kong/docs.konghq.com.git" \
+          "or Ctrl-C to cancel."
+
+  set -e
+  git add app/_data/kong_versions.yml
+  git commit -m "chore(*) update release metadata for $version"
+
+  git push --set-upstream origin "$branch"
+  hub pull-request -b main -h "$branch" -m "Release: $version" -l "pr/please review,pr/do not merge"
+
+  SUCCESS "Make sure you give Team Docs a heads-up" \
+          "once the release is pushed to the main repo." \
+          "When the main release PR is approved, you can proceed to:" \
+          "    $0 $version merge"
+}
+
+#-------------------------------------------------------------------------------
+function submit_release_pr() {
+  base=$1
+  version=$2
+
+  if ! git log -n 1 | grep -q "release: $version"
+  then
+    die "Release commit is not at the top of the current branch. Did you commit the version bump?"
+  fi
+
+  git log
+
+  CONFIRM "Press Enter to push the branch and open the release PR" \
+    "or Ctrl-C to cancel."
+
+  set -e
```

Comment:
an overall comment on the `set -e` and `SUCCESS` interaction. A nicer and more userfriendly way would be to use traps. This maybe out of scope for PR but I'll leave it here for inspiration.

Reply:
I admit that I have never used `trap` in such a way (I try to avoid bash script when I can ...).

From what I have read, it seems that we would *still* need `set -e` (to produce the signal that then would be captured by trap).

Let's call it out of scope for now, it seems a bit ... philosophical as well (one global rescue function vs lots of small traps?)

Reply:
We had a live conversation about this.

@gszr , I think there must be a particular reason why `set -e` is set/unset on every action instead of setting it once globally. Is there? What is your opinion of using `trap` to recover? At the very least we should be able to print "the command blablabla failed with blebleble"